src/merkle/empty_roots.rs

```diff
@@ -1,6 +1,12 @@
-use super::{Felt, RpoDigest, WORD_SIZE, ZERO};
+use super::{Felt, RpoDigest, Word, WORD_SIZE, ZERO};
 use core::slice;
 
+// CONSTANTS
+// ================================================================================================
+
+/// A word consisting of 4 ZERO elements.
+pub const EMPTY_WORD: Word = [ZERO; WORD_SIZE];
```

Comment:
Had you considered using `Word::default()` instead?

Reply:
I thought about it, but since `default()` isn't `const fn` I wasn't sure how the compiler would treat it - so, decided to use a constant as it can be potentially optimized better.